src/NServiceBus.Core.Tests/API/APIApprovals.ApproveNServiceBus.approved.txt

```diff
@@ -975,6 +975,13 @@ namespace NServiceBus
     {
         public static NServiceBus.SecondLevelRetries.Config.SecondLevelRetriesSettings SecondLevelRetries(this NServiceBus.EndpointConfiguration config) { }
     }
+    public class SecondLevelRetryContext
+    {
+        public SecondLevelRetryContext() { }
+        public System.Exception Exception { get; set; }
+        public NServiceBus.Transports.IncomingMessage Message { get; set; }
+        public int SecondLevelRetryAttempt { get; set; }
```

Comment:
The first sentence is a valid and good argument. I don't see the "adds value" part in the second sentence. How does prefixing a property with a `SecondLevelRetry` in the context of a second level retry setting under a second level retry context provide any value? I fail to see this. I'd just name this `Attempt` then